hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -679,6 +679,42 @@ public void deleteSavepoint(String savepointTime) {
     logger.info("Savepoint " + savepointTime + " deleted");
   }
 
+  /**
+   * Delete a compaction request that is pending.
+   *
+   * NOTE - This is an Admin operation.
+   * With async compaction, this is expected to be called with async compaction and ingestion shutdown.
+   * Otherwise, async compactor could fail with errors
+   *
+   * @param compactionTime - delete the compaction time
+   * @return
+   */
+  public void deletePendingCompaction(String compactionTime) {
```

Comment:
Is this a tool ? We should move this to hoodie-cli

Reply:
I have made it a private method. Its used for the rollback case